projects/npm-tools/packages/npm-scripts/src/utils/runWebpackAsBundler.js

```diff
@@ -3,16 +3,12 @@
  * SPDX-License-Identifier: BSD-3-Clause
  */
 
-/* eslint-disable @liferay/no-dynamic-require */
+/* XXeslint-disable @liferay/no-dynamic-require */
```

Comment:
XX ?

Reply:
Oops, that must be an error. I didn't type that on purpose...

Reply:
https://github.com/liferay/liferay-frontend-projects/pull/1088